kerl

```diff
@@ -281,6 +261,9 @@ KERL_BUILD_DOCS=
 KERL_DOC_TARGETS=chunks
 KERL_BUILD_BACKEND=
 KERL_RELEASE_TARGET=
+"
+act_on_kerl_cfgs "$_KERL_CFGS" "cache"
```

Comment:
Called it here to have the `_KERL_CFGS` variable defined, but it does (similar to) what's above, from lines 227-268.